.github/workflows/build-push.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and Push
+on: push
+
+env:
+  NODE_VERSION: 18
+  JAVA_VERSION: 11
+  JAVA_DISTRIBUTION: 'adopt'
+
+jobs:
+  build-push:
+    name: run tests, build and push
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Node ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'npm'
+
+      - name: Set up JDK 11
```

Comment:
where do we use java?

Reply:
im not sure why we are setting up java, i was worried about removing it

Reply:
ill leave for now, but take an action item to look into it